### PR TITLE
Initial commit of retinotopy code and stimuli

### PR DESCRIPTION
**Pull request by CameronTEllis**
Initial pull request for Retinotopy scripts and stimuli

### CODE REVIEW

File: Scripts/GenerateTrials_Retinotopy.m [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    
+    Parameters.Stim_Repetitions = 8;      % How many times does each stim repeat?
+    
+    Parameters.cut_out=       0;           % cut out n deg (radius) of center
+    
+    Parameters.cut_in=       40;           % cut in n deg (radius) from outer max; equals max eccen of wedge
+    
+    Parameters.fmask_size=   1.5;              % What is the width of the lines?
+    
+    Parameters.NumRepetitions = 1;  %One repetition of each
```
Comment (tristansyates) [inline Scripts/GenerateTrials_Retinotopy.m:90]: redundant -- defined above the if/elseif

File: Scripts/Experiment_Retinotopy.m [hunk 2]
```diff
@@ -0,0 +1,758 @@
...
+    grow =0;
+    prevTime = GetSecs - start_time;
+end
+
+if FixationStimulus==1
+    %When does the movie start
+    Data.Timing.movieStartIndex = round(rand(1) * (MovieLength - (rot_period + Redundancy + 5))); % Add a bit of a buffer
+    
+    %If the movie is too short then reload the default one
+    if sign(Data.Timing.movieStartIndex)==-1
```
Comment (tristansyates) [inline Scripts/Experiment_Retinotopy.m:346]: don't quite understand what's going on here, isn't this just pulling the same moviename from parameters?

**Review by tristansyates — APPROVED** [review]
Learned a bit more of the history of the project here! The frankenstein scripts aren't actually hard to follow at all, I think its all looking good!

File: Scripts/GenerateTrials_Retinotopy.m (hunk 1 above)
Comment (CameronTEllis) [reply Scripts/GenerateTrials_Retinotopy.m:90]: Kind of but not exactly. It is only reset again in the if statement if `TravellingWave` == 0, since it should only be one repetition for that condition.

File: Scripts/Experiment_Retinotopy.m (hunk 2 above)
Comment (CameronTEllis) [reply Scripts/Experiment_Retinotopy.m:346]: The idea is that the movie could start at a certain point but if there isn't enough movie to complete the block then it will revert to a default where there definitely is enough movie
